Common/Orders/BracketOrder.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using QuantConnect.Interfaces;
+using QuantConnect.Securities;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Orders
+{
+    public class BracketOrder : Order
```

Comment:
I think these new order types should be added in a separate PR

Reply:
Once they are functional will bring them out into a new PR @StefanoRaggi